Src/BootCamp.Chapter/Area51.cs

```diff
@@ -0,0 +1,204 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace BootCamp.Chapter
+{
+    public class Area51
+    {
+        private List<Credentials> loggedInAccounts = new List<Credentials>();
+        readonly CredentialsManager credentialsManager = new CredentialsManager("Login.txt");
+        const string login = "login";
+        const string create = "create";
+        const string see = "see";
+        const string quit = "quit";
+        public void Demo()
+        {
+            string answer = string.Empty;
+            while (answer != quit)
+            {
+                answer = GetAnswerMainMenu();
+                GoToPartOfProgram(answer);
+            }
+        }
+        private string GetAnswerMainMenu()
+        {
+
+            string input = string.Empty;
+            bool isLoopMainMenu = true;
+
+            while (isLoopMainMenu)
+            {
+                Console.Clear();
+                Console.WriteLine("Login\r\nCreate\r\nSee\r\nQuit");
```

Comment:
prefer to use ``Environemnt.NewLine`` over \r\n because that's just for windows.